app/assets/stylesheets/new_board/_story.scss

```diff
@@ -11,6 +14,10 @@
       color: white !important;
     }
 
+    &--focused {
+      background-color: #d8d84a !important;
```

Comment:
Why was `!important` necessary here? Also, it would be good to extract this color to a variable :smile:

Reply:
Oh without the `!important` the background will not to be replaced, because of class `--collapsed`. I will create a css variable.